src/components/application_manager/test/application_impl_test.cc

```diff
@@ -507,19 +507,19 @@ TEST_F(ApplicationImplTest, IsCommandLimitsExceeded_LimitFromPT) {
 
   EXPECT_CALL(policy_interface, GetNotificationsNumber(_))
       .WillOnce(Return(100u));
```

Comment:
@okozlovlux what is this number 100u supposed to mean? Shouldn't it be declared with a variable?